editor/state-machine/inputs.mdx

```diff
@@ -3,7 +3,13 @@ title: 'Inputs'
 description: ''
 ---
 
-Inputs are the contract between designers and developers. As Designers, we use Inputs as a way to control the transitions in our state machine by assigning them as conditions. Developers tie into the inputs at runtime and define conditions through code that can change those inputs. 
+<Warning>
+  We strongly recommend controlling your Rive file through [Data Binding](../data-binding), rather than Inputs. Not only is Data Binding more powerful, it's easier to [control via code at runtime](../../runtimes/data-binding).
```

Comment:
Can you update all the links to be root-relative? You can read here on Mintilfy docs why: https://mintlify.com/docs/text#linking-to-pages

Reply:
I updated all the links related to this PR. I'm going to do all of the others in this repo, but there are a ton so I want to do it in a separate PR.